enterprise/frontend/src/embedding/data-picker/saved-entity-picker/SavedEntityPicker.jsx

```diff
@@ -46,7 +46,6 @@ const ALL_PERSONAL_COLLECTIONS_ROOT = {
 };
 
 function SavedEntityPicker({
-  type,
```

Comment:
This value can now only be `model` so I hardcode it in the component body instead. Please see the comment on `DataSeletor.jsx`'s `getCardType()` function